chrome_manifest_v3/class_app.js

```
@@ -0,0 +1,848 @@
+   */
+  bookend(bookend, text, style) {
+    return (
+      '<' +
+      bookend +
+      (style ? ' style="' + style + '"' : '') +
+      '>' +
+      (text || '') +
+      '</' +
+      bookend +
+      '>'
+    );
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)